config.toml

```diff
@@ -32,3 +32,6 @@ pygmentsstyle = "monokailight"
 
 [security.funcs]
   getenv = ['^HUGO_', 'BRANCH']
+
+[build.environment]
+  HUGO_VERSION = "v0.92.2"
```

Comment:
I'm not sure this is the correct file to place this. Do you remember where you sourced it from? I presumed it might be [`netlify.toml`](https://docs.netlify.com/integrations/frameworks/hugo/#hugo-version) or something.

Reply:
TBH, I'm not sure either. Should I move it to `netlify.toml`? Also not sure how we can double check which hugo version is really used on netlify then.